index.js

```diff
@@ -7,26 +7,7 @@ import esbuild from 'esbuild';
  * @typedef {import('esbuild').BuildOptions} BuildOptions
  */
 
-const ssrFunctionRoute = '/api/__render';
-
-const functionJson = `
-{
-	"bindings": [
-		{
-			"authLevel": "anonymous",
-			"type": "httpTrigger",
-			"direction": "in",
-			"name": "req",
-			"route": "__render"
-		},
-		{
-			"type": "http",
-			"direction": "out",
-			"name": "res"
-		}
-	]
-}
-`;
+const ssrFunctionRoute = '/api/sk_render';
```

Comment:
The docs need to be updated to mention that this route is now reserved https://github.com/derkoe/svelte-adapter-azure-swa/blob/7a855e8885385ba12a8a71f603dc6f5e8537b732/README.md#L122-L123